packages/gatsby/src/commands/recipes.ts

```diff
@@ -0,0 +1,58 @@
+import telemetry from "gatsby-telemetry"
+import execa from "execa"
+import path from "path"
+import fs from "fs"
+import detectPort from "detect-port"
+
+module.exports = async (program: IProgram): Promise<void> => {
+  const recipe = program._[1]
+  // We don't really care what port is used for GraphQL as it's
+  // generally only for code 2 code communication or debugging.
+  const graphqlPort = await detectPort(4000)
+  telemetry.trackCli(`RECIPE_RUN`, { recipe })
```

Comment:
This should match the telemetry schema. Eg there is no recipe field, but there is name.

```suggestion
  telemetry.trackCli(`RECIPE_RUN`, { name: recipe })
```